plugins/woocommerce/templates/order/customer-history.php

```diff
@@ -16,9 +16,9 @@
 /**
  * Variables used in this file.
  *
- * @var int   $order_count   The number of paid orders placed by the current customer.
- * @var float $total_spent   The total money spent by the current customer.
- * @var float $average_spent The average money spent by the current customer.
+ * @var int   $orders_count   The number of paid orders placed by the current customer.
```

Comment:
This is actually interesting. I know that you are updating the labels based on what the API returns so we don't really have influence over that. I was wondering if there is a difference between this two versions. ChatGPT helped:

> - Use "order_count" if you are referring to a single count or quantity of orders. This implies that you are counting the total number of orders.
> 
> - Use "orders_count" if you are referring to a count or quantity associated with multiple orders. This could mean that you have a count for each order or that you are counting something related to multiple orders, such as the total count of items across all orders.

This makes it sound like the version we had was more suitable.

This is just an of topic :) not really part of the review.